maxim/maxim.py

```
@@ -256,19 +316,48 @@ def __syncFolders(self):
+
+        Args:
+            entity (str): The entity type (e.g., "PROMPT", "FOLDER", "PROMPT_CHAIN").
+            id (str): The entity ID.
+
+        Returns:
+            str: The formatted cache key.
+        """
         if entity == "PROMPT":
             return f"prompt:{id}"
-        else:
+        elif entity == "PROMPT_CHAIN":
```

Ruff (0.11.9)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)